vnext/Desktop.DLL/packages.config

```diff
@@ -9,5 +9,4 @@
   <package id="ReactNative.V8Jsi.Windows" version="0.64.8" targetFramework="native" />
   <package id="ReactWindows.ChakraCore.ARM64" version="1.11.20" targetFramework="native" developmentDependency="true" />
   <package id="ReactWindows.OpenSSL.StdCall.Static" version="1.0.2-p.5" targetFramework="native" />
-  <!-- package id="ReactNative.Hermes.Windows" version="0.7.2" targetFramework="native" / -->
 </packages>
```

Comment:
FWIW, this commented line is here to enable quickly producing a Hermes-compatible build of RNW32.dll to check status, etc.

Reply:
The current NuGet package only includes the UWP flavor. Think it might make sense to produce multiple NuGet packages in the future, then enable win32 here once that is done.